articles/machine-learning/reference-yaml-mltable.md

```diff
@@ -0,0 +1,144 @@
+---
+title: 'CLI (v2) mltable YAML schema'
+titleSuffix: Azure Machine Learning
+description: Reference documentation for the CLI (v2) MLTable YAML schema.
+services: machine-learning
+ms.service: machine-learning
+ms.subservice: mltable
+ms.topic: reference
+ms.custom: cliv2, event-tier1-build-2022
+
+author: xunwan
+ms.author: xunwan
+ms.date: 09/15/2022
+ms.reviewer: s-polly
+---
+
+# CLI (v2) data YAML schema
+
+[!INCLUDE [cli v2](../../includes/machine-learning-cli-v2.md)]
+
+The source JSON schema can be found at https://azuremlschemas.azureedge.net/latest/MLTable.schema.json.
+
+
+
+[!INCLUDE [schema note](../../includes/machine-learning-preview-old-json-schema-note.md)]
+
```

Comment:
Should we have a section at the beginning "When to use MLTable" e.g. "in cases of complex schema, automl. If cases don't suit or you want to write your own parsing logic in python then please use uris?"